<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>URLcheck</class>
 <widget class="QMainWindow" name="URLcheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>URLcheck</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Avaa tiedosto</string>
    </property>
   </widget>
   <widget class="QTableView" name="urltable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
    <property name="asd" stdset="0">
     <string/>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="urlButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Avaa Url</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>http://</string>
    </property>
   </widget>
   <widget class="QLabel" name="urlLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>URL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="aboutLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>About:</string>
    </property>
   </widget>
   <widget class="QLabel" name="aboutLabel2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tarkastaa annetun hyperteksti tiedoston</string>
    </property>
   </widget>
   <widget class="QLabel" name="aboutLabel3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>tai osoitteen sisältämät http-linkit.</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="methodBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GET</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HEAD</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="methodLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HTTP-pyyntömetodi</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
